src/zulip.rs

```diff
@@ -298,7 +298,7 @@ async fn workqueue_commands(
                 pluralize("PR", assigned_prs.len())
             );
             writeln!(response, "Review capacity: {capacity}\n")?;
-            writeln!(response, "*Note that only selected PRs that are assigned to you are considered as being in the review queue.*")?;
+            writeln!(response, "*Note that only some PRs that are assigned to you are considered as being in the review queue.*")?;
```

Comment:
While we are at it, let's make it clear that we are talking about the user review queue.
```suggestion
            writeln!(response, "*Note that only some PRs that are assigned to you are considered as being in your review queue.*")?;
```
or maybe even this restructured variant,
```suggestion
            writeln!(response, "*Note that only certain PRs assigned to you are included in your review queue.*")?;
```

Reply:
Reworded, thanks.